src/sql/magic_cmd.py

```diff
@@ -68,12 +69,15 @@ def _validate_execute_inputs(self, line):
             f"Valid commands are: {', '.join(AVAILABLE_SQLCMD_COMMANDS)}"
         )
 
+        ALLOWED_DUPLICATES = []
+
         if line == "":
             raise exceptions.UsageError(VALID_COMMANDS_MSG)
         else:
             # directly use shlex since SqlCmdMagic does not use magic_args from parse.py
             split = shlex.split(line, posix=False)
             command, others = split[0].strip(), split[1:]
+            check_duplicate_arguments(self.execute, "sqlcmd", split, ALLOWED_DUPLICATES)
```

Comment:
Instead of passing an empty list here you can declare default value of allowed_duplicates like:
```
def check_duplicate_arguments(
    magic_execute, cmd_from, args, allowed_duplicates=[]
) -> bool:
```

Reply:
Fixed [here](https://github.com/ploomber/jupysql/pull/908/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R160-R163)